src/runtime/security/meta_access_controller.cpp

```diff
@@ -0,0 +1,57 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "meta_access_controller.h"
+
+#include <dsn/tool-api/rpc_message.h>
+#include <dsn/utility/flags.h>
+#include <dsn/tool-api/network.h>
+
+namespace dsn {
+namespace security {
+DSN_DEFINE_string("security",
+                  rpc_code_white_list,
+                  "",
+                  "white list of rpc codes for meta_access_controller");
```

Comment:
Why it's a user specified parameter?

Reply:
I make `rpc_code_white_list` configurable. so if we should only change the config file, but not the cpp file if we want to add a rpc code to this white list.